ci-operator/config/redhat-et/microshift/redhat-et-microshift-main.yaml

```diff
@@ -18,7 +18,7 @@ build_root:
     tag: rhel-8-release-golang-1.17-openshift-4.10
 images:
 - dockerfile_path: packaging/images/microshift/Dockerfile
-  to: linux-amd64-4.8.0-0.microshift
+  to: linux-amd64-4.10.0-0.microshift
```

Comment:
This tag is referenced by at least one step ref I believe, so it'll need to be updated as well.  Maybe we can drop the versioning from the tag to avoid the need to update them in the future.